network/hostapd-mana/files/.svn/entries

```diff
@@ -0,0 +1,368 @@
+10
```

Comment:
Are these .svn files necessary? Seems like bloat from the barrier breaker days.

Reply:
Other than that, everything is looking good I think.

Reply:
I've removed them. Shouldn't be needed at all.